packages/core/src/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const InjectionSymbols = {
-  MessageRepository: Symbol('MessageRepository'),
+  MessagePickupRepository: Symbol('MessagePickupRepository'),
```

Comment:
I agree. However I think I'll need to update a bit the logic in order to avoid dependency cycles between `MessageSender` and `MessagePickupApi`. Probably adding some parameter to `sendMessage` or even triggering an internal event that can be listened in Message Pickup.